src/screens/auth/LoginScreen.tsx

```diff
@@ -28,6 +30,9 @@ const LoginScreen = () => {
                 navigation.navigate(routes.PHONE_SCREEN);
               }}
             />
+            {Platform.OS=== "android" ?
+             <Text style={styles.text}>{ANDROID_VERSION}</Text>
+             :<Text style={styles.text}>{ANDROID_VERSION}</Text> }    
```

Comment:
aqui no estas usando la version de iOS

Reply:
tambien podrias hacerlo mas conciso si haces el if dentro del `Text`

```suggestion
             <Text style={styles.text}>{Platform.OS=== "android" ? ANDROID_VERSION : IOS_VERSION }</Text>
```